Promact.Oauth.Server/src/Promact.Oauth.Server/Startup.cs

```diff
@@ -27,9 +27,10 @@ namespace Promact.Oauth.Server
     public class Startup
     {
         private ILoggerFactory _loggerFactory { get; }
-
+        private IHostingEnvironment CurrentEnvironment { get; set; }
```

Comment:
Ok, in that case, naming convention is wrong. private variables are always camel cased with underscore.